doc/enterprise/overview.md

```diff
@@ -0,0 +1,30 @@
+# Overview
+
+![alt tag](enterprise.png)
+
+Pachyderm Enterprise Edition includes everything you need to scale and manage Pachyderm data pipelines in an enterprise setting. It is secure and simple, and it delivers the most recent version of Pachyderm along with:
+
+- Administrative and security features needed for enterprise-scale implementations of Pachyderm 
+- Visual and interactive interfaces to Pachyderm  
```

Comment:
stats should be on this list. "Detailed job statistics for faster development and data insight"